cities.json

```diff
@@ -5564,6 +5564,14 @@
                         "right": "-67.004"
                     }
                 },
+                "palmas_brazil": {
+                    "bbox": {
+                    	"top": "-10.087",
+                    	"bottom": "-10.464",
+                    	"left": "-48.525",
+                    	"right": "-48.144",
```

Comment:
The trailing comma makes this an invalid json file. You can verify if your file is valid in online tools such as http://jsonlint.com. See merge and fix here https://github.com/mapzen/metroextractor-cities/commit/76305ff60c9219d8e7568ffb7e20de6ab086cb66